### PR TITLE
v0.1.0-alpha is no longer a branch, switched this staging

### PR DESCRIPTION
**Pull request by EricSchles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Assumptions:
 
 4. Switch to the alpha release version of the code. This is the latest stable release.
```
Comment (nmonga91) [inline README.md]: we should reword this line since there is no "alpha" release anymore.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Assumptions:
...
 
-        $ git checkout v0.1.0-alpha
+        $ git checkout staging
```
Comment (nmonga91) [inline README.md]: The latest stable release would be `master` as opposed to `staging`. Line 33 down below instructs the user to refer to staging for the latest, work-in-progress features.

Comment (EricSchles) [inline README.md]: sounds good.  Made the change.

File: README.md (hunk 1 above)
Comment (nmonga91) [inline README.md]: This could say "Switch to the master branch of the code. This is the latest stable release."

**Pull request closed without merging**